typescript/libs/view_helpers.py

```diff
@@ -213,10 +229,28 @@ def reload_buffer(view, client_info=None):
             client_info.change_count = info.modify_count
         client_info.pending_changes = False
 
+def reload_buffer_on_worker(view):
```

Comment:
The ``reload_buffer`` send reload commands to both the main server and the worker server (if up); and ``reload_buffer_on_worker`` only sends reload commands to the worker server. Therefore syncing shouldn't be a problem. And I cannot repro the problem you mentioned, the error message would go away as expected on the panel, though it may take a second.